store/migration/10001__init_schema.sql

```diff
@@ -1300,3 +1300,39 @@ WHERE
     rowid = old.rowid;
 
 END;
+
+-- Deployment Configuration.
+-- deployment_config stores deployment configurations at project level.
+CREATE TABLE deployment_config (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    row_status TEXT NOT NULL CHECK (
+        row_status IN ('NORMAL', 'ARCHIVED')
+    ) DEFAULT 'NORMAL',
+    creator_id INTEGER NOT NULL REFERENCES principal (id),
+    created_ts BIGINT NOT NULL DEFAULT (strftime('%s', 'now')),
+    updater_id INTEGER NOT NULL REFERENCES principal (id),
+    updated_ts BIGINT NOT NULL DEFAULT (strftime('%s', 'now')),
+    project_id INTEGER NOT NULL REFERENCES project (id),
```

Comment:
shall we enforce uniqueness since for now we only support single config per project?